payments/models.py

```diff
@@ -40,6 +41,7 @@
 pm = get_plugin_manager()
 
 
+@reversion.register(exclude=("created_on", "updated_on"))
 class Discount(TimestampedModel):
```

Comment:
Didn't notice this in earlier revisions/etc. but this actually also needs a unique constraint on the discount code. I was able to create duplicate codes. (For simplicity this should just unique on the code and not the code and integrated system.)